notebooks/238-dolly-2-instruction-following/238-dolly-2-instruction-following.ipynb

```diff
@@ -0,0 +1,557 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ef2ed242-3561-464c-8d1c-cc3862e23702",
+   "metadata": {},
+   "source": [
+    "# Instruction following using Databricks Dolly 2.0 and OpenVINO\n",
+    "\n",
+    "The instruction following is one of the cornerstones of the current generation of large language models(LLMs). Reinforcement learning with human preferences ([RLHF](https://arxiv.org/abs/1909.08593)) and techniques such as [InstructGPT](https://arxiv.org/abs/2203.02155) has been the core foundation of breakthroughs such as ChatGPT and GPT-4. However, these powerful models remain hidden behind APIs and we know very little about their underlying architecture. Instruction-following models are capable of generating text in response to prompts and are often used for tasks like writing assistance, chatbots, and content generation. Many users now interact with these models regularly and even use them for work but the majority of such models remain closed-source and require massive amounts of computational resources to experiment with.\n",
```

Comment:
```suggestion
    "Instruction-following is one of the cornerstones of the current generation of large language models (LLMs). Reinforcement Learning with human preferences ([RLHF](https://arxiv.org/abs/1909.08593)) and techniques such as [InstructGPT](https://arxiv.org/abs/2203.02155) have been at the core foundation of breakthroughs with models like GPT-4. However, these powerful models are often hidden behind APIs and we know very little about their underlying architecture. Instruction-following models are capable of generating text in response to prompts and are often used for tasks like writing assistance, chatbots and content generation. While many users are now interacting with these models regularly and even use them for work, the majority remain closed-source and require massive amounts of computational resources to experiment with.\n",
```
Using "instruction-following" to be consistent (you had two versions). I don't think we have permission to use the name ChatGPT. If you have a research paper or something we can refer to about ChatGPT it might be OK.